cfgov/jobmanager/tests/models/test_blocks.py

```diff
@@ -10,13 +11,15 @@
 from jobmanager.models.panels import GradePanel, RegionPanel
 
 
-def make_job_listing_page(title, close_date, grades=[], regions=[]):
+def make_job_listing_page(title, close_date=None, grades=[], regions=[],
+                          **kwargs):
     page = mommy.prepare(
         JobListingPage,
         title=title,
-        close_date=close_date,
+        close_date=close_date or timezone.now().date(),
```

Comment:
What value does this have at the end of this line? 

close_date or [value here] <= this looks like a boolean output with a date. Doesn't make sense to me.